### DISCUSSION TITLE
CDatataable multi-column sorting

### DISCUSSION THREAD

**Discussion by HechmiNada** [coreui/coreui-free-vue-admin-template#274] [General]
Hello

I'm on CoreUI for Vue.js and I'm using the CDatatable component and wanted to do multi-column sorting (i.e. order by a specific column and then a secondary column if the data in the first column is identical)
For example: when I click on column index 0, the data will be sorted by column index 0, and then sub-sorted by column index 1

How can i do it ?

**Comment by pervej-fac** [comment]
Code released under [the MIT license](https://github.com/coreui/coreui-free-react-admin-template/blob/master/LICENSE). There is only one limitation you can't can’t re-distribute the CoreUI as stock. You can’t do this if you modify the CoreUI. In past we faced some problems with persons who tried to sell CoreUI based templates.

here is shown, we can not re-distribute. what is meant by this?

Suppose I have used the dashboard in my sales management software and others. and Can i sell the softwares? Have any restriction by this re-distribution issues?
